Gemfile

```diff
@@ -63,6 +63,7 @@ gem 'will_paginate-bootstrap'
 gem 'yt'
 gem 'yui-compressor'
 gem 'youtube_rails'
+gem 'google-api-client', '0.8'
```

Comment:
@lokeshh I'll defer to @tansaku.  I would rather see it updated with latest ruby api, or converted to strictly using just youtube api.